src/mpcCoreKit.ts

```diff
@@ -759,12 +774,13 @@ export class Web3AuthMPCCoreKit implements ICoreKit, IMPCContext {
    * Throws an error if signature type is not ed25519.
    */
   public getPubKeyEd25519(): Buffer {
-    if (this._sigType !== "ed25519") {
-      throw CoreKitError.default(`getPubKeyEd25519 not supported for signature type ${this.sigType}`);
+    if (!this.supportedSigTypes.has(KeyType.ed25519)) {
+      throw CoreKitError.default(`ed25519 is not supported, please configure tssLib with ed25519 signature type `);
     }
 
-    const tssCurve = getKeyCurve(this.keyType);
-    const p = this.getPubKeyPoint().toEllipticPoint(tssCurve);
+    const tssCurve = getKeyCurve(KeyType.ed25519);
+    // ed25519 only support account index 0 ?
+    const p = this.getPubKeyPoint(KeyType.ed25519, 0).toEllipticPoint(tssCurve);
```

Comment:
```suggestion
    const p = this.getPubKeyPoint(KeyType.ed25519).toEllipticPoint(tssCurve);
```
i don't think it's a good idea to just assume index 0 here.
better throw an error if the user is on unsupported account index. alternatively, simply return the result.